setup.cfg

```diff
@@ -9,7 +9,7 @@ license_file = LICENSE
 
 [tool:pytest]
 minversion = 3.0
-testpaths = tests
+testpaths = tests examples
```

Comment:
The reason this is set in tox is because I want people to be able to run `pytest` without having to install the examples.

Reply:
I see.. `{posargs:tests examples}` in tox.ini then?

Reply:
updated